local.properties

```diff
@@ -4,5 +4,6 @@
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-#Thu May 17 17:18:01 SGT 2018
-sdk.dir=/Users/visenze/Library/Android/sdk
+#Mon Aug 20 19:14:16 SGT 2018
+ndk.dir=/Users/visenze 1/Library/Android/sdk/ndk-bundle
```

Comment:
@yulu 

just to check, is this /Users/visenze or "/Users/visenze 1" ?

Reply:
I will discard the of this file. this is my local path